tools/docker/ts-integration.Dockerfile

```diff
@@ -21,11 +21,7 @@ WORKDIR ${SRCROOT}
 
 # copy over all our dependencies
```

Comment:
You can drop this dockerfile altogether, it was used for cypress integration tests which we removed